test/e2e/describe_test.go

```diff
@@ -358,42 +417,56 @@ content:
         some.other.annotation: some other value
       image: %s%s
       imageType: Image
+      layers:
+      - %s
```

Comment:
In text output it will look same while yaml output will have what there in e2e test. Yaml format is not we are controlling manually. This one we get when we do marshalling of struct https://github.com/carvel-dev/imgpkg/blob/develop/pkg/imgpkg/cmd/describe.go#L194.

Reply:
updated